document/js-api/index.bs

```diff
@@ -548,6 +550,9 @@ interface Module {
 };
 </pre>
 
+Per ECMA-262 requirements for source phase import integration, the interface object for {{Module}} must have as its [[Prototype]] be %AbstractModuleSource%, created as if by ObjectCreate(%AbstractModuleSource%), instead of %ObjectPrototype%.
+In addition, the interface prototype object for {{Module}} must have as its [[Prototype]] %AbstractModuleSource.prototype%, created as if by ObjectCreate(%AbstractModuleSource%), instead of %ObjectPrototype%.
```

Comment:
Is this the same wording that is already used in other specs to set a non-WebIDL prototype? (@lucacasonato you found an example of that, right?)

Reply:
The other option here would be a WebIDL attribute of some kind, but it seems like this would require WebIDL changes, see discussion in https://github.com/whatwg/webidl/issues/1322. We could possibly align with an approach like that if it is deemed appropriate, but we also shouldn't speculate.